module-4/backend/week_one.md

```diff
@@ -5,15 +5,44 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+getting used to the new syntax
+
 2. What are some tools to help debug JavaScript code?
+
+dedugger, console.log
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+mocha, chai. ive heard of jasmine too but i havent tried it out yet
+
 4. What is the syntax for invoking a function? Give an example.
+
+function()
+
 5. What's `this` in JavaScript?
+
+what are currently in. like an object or an event. when you click on something, this is now what you are inside of
+
 6. What is Webpack and why is it useful?
+
+its like rails server. compiles your code and spins up a local server for you to check your code as you code
+
 7. When/why do you want to use event delegation?
+
+
+
 8. What's `npm` and what do we use it for?
 
+node package manager. its used as a command to install something through your terminal like a gem in ruby
```

Comment:
It IS used as a command, and you want to make sure you know that... 
npm is used to add libraries and packages to a node project. We use it to specify the packages that are required in the project, and we also use it to build the project before deployment.